src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/ConfirmArrivalDetail.kt

```diff
@@ -23,6 +23,16 @@ data class ConfirmArrivalDetail(
   @field:Pattern(regexp = "^([0-9]{2}|[0-9]{4})/[0-9]+[a-zA-Z]$", message = "PNC is not valid")
   val pncNumber: String? = null,
 
+  @Schema(
+    description = "The offender's Prison number.",
+    example = "A1234AA",
+    pattern = "^[A-Za-z]\\d{4}[A-Za-z]{2}\$",
+    maxLength = 7
+  )
+  @field:Length(max = 7)
+  @field:Pattern(regexp = "^[A-Za-z]\\d{4}[A-Za-z]{2}\$", message = "Prison number is not valid")
+  val prisonNumber: String? = null,
```

Comment:
Not actually used yet but is required to allow users to select which of a many matches refers to this arrival